pkg/resources/nodeportproxy/nodeportproxy.go

```diff
@@ -415,9 +415,11 @@ func FrontLoadBalancerServiceCreator(data *resources.TemplateData) reconciling.N
 				}
 			}
 
+			// set of Source IP ranges
+			sourceIPList := sets.String{}
 			if data.Seed().Spec.NodeportProxy.Envoy.LoadBalancerService.SourceRanges != nil {
 				for _, cidr := range data.Seed().Spec.NodeportProxy.Envoy.LoadBalancerService.SourceRanges {
-					s.Spec.LoadBalancerSourceRanges = append(s.Spec.LoadBalancerSourceRanges, string(cidr))
+					sourceIPList.Insert(string(cidr))
```

Comment:
This misses being written to `s.Spec.LoadBalancerSourceRanges`, I believe.

Reply:
In addition, is this the only place where `s.Spec.LoadBalancerSourceRanges` can be set?